projects/vdk-plugins/vdk-notebook/README.md

```diff
@@ -0,0 +1,28 @@
+# notebook
+
+A VDK plugin for working with notebooks.
+
```

Comment:
The PR description contains additional information that is already know:

> Adding a new plugin which will support running data jobs which consists of .ipynb files.

Maybe add it to the README